packages/brokeneck-react/src/components/Groups.js

```diff
@@ -54,7 +106,7 @@ export default function Groups() {
     currentToken,
     useUpdateToken,
     useTablePagination
-  } = usePagination({ pageSizeOptions: [2, 3, 4] }) // TODO: temp small page sizes for testing
```

Comment:
this override is still useful as we're still having some issues with pagination. I know it's not great